docs/monitoring/metrics.md

```diff
@@ -1236,42 +1351,42 @@ Certain RocksDB native metrics are available but disabled by default, you can fi
     <tr>
       <th rowspan="12"><strong>Task</strong></th>
       <td>numBytesInLocal</td>
-      <td>The total number of bytes this task has read from a local source.</td>
+      <td>The total number of bytes this task has read from a local source. Deprecated: see NettyShuffle.</td>
```

Comment:
let's remove the description entirely and put a nice `attention` icon before the deprecation notice.